### PR TITLE
Design

### PR DESCRIPTION
**Pull request by hemilrajpura-simform**
Design is completed

### CODE REVIEW

**Comment by hemilrajpura-simform** [comment]
Screenshot from 2022-01-05 09-53-52

File: index.html [hunk 1]
```diff
@@ -1,18 +1,164 @@
...
+          </button>
+        </div>
+      </div>
+    </div>
 
 <script>
 
-    
-</script>
+var screen = document.querySelector('.screen');
```
Comment (abhie-sheik) [inline index.html]: Move this JS code in a separate file named app.js or script.js and load that file here.

Comment (hemilrajpura-simform) [reply index.html]: okay

File: app.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+
+
+
+function Function() {
+  var e = document.getElementById("Function").value;
+  console.log(e);
+
+  if (e == 1) {
```
Comment (abhie-sheik) [inline app.js]: Use switch statement here instead of if, else if..

Comment (hemilrajpura-simform) [reply app.js]: I did used switch case there but due to some error it was not working so i used if statement, but i will use switch case and resolve the error, Thank you for pointing out.

Comment (abhie-sheik) [reply app.js]: As discussed try once again and verify your syntax, and update the PR.

Comment (hemilrajpura-simform) [reply app.js]: Updated the changes

**Review by abhie-sheik — APPROVED** [review]

**Pull request merged**